src/components/general/modals/share-experiences-modal.js

```diff
@@ -20,6 +62,43 @@ export default function ShareExperiencesModal() {
     }
   }
 
+  function handleClose() {   
+    toggleModal(false);
+    setExperiences({
+      edited: false,
+      positive: false, 
+      tags: [] 
+    });
+    setCurrentScreen(0);
+  }  
+
+  async function handleSubmit(){   
```

Comment:
async keyword is only needed when you are using await keyword inside the function. here, you are using .then, so you have two options. either move that code to use await, or just remove async keyword. this wont break the code but its superfluous